.github/workflows/behat-stability-1.yml

```diff
@@ -0,0 +1,63 @@
+name: 'Behat stability 1'
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push or pull request events but only for the main branch
+  push:
+    branches: [ main ]
+  pull_request: { }
+
+  # @todo Enable after successful testing
+  # Allows you to run this workflow manually from the Actions tab
+  # workflow_dispatch:
+
+
+# github.head_ref is only set when the workflow was triggered by a pull_request and it contains the value of the source branch of the PR.
+# github.ref_name will than only be used if the workflow was not triggered by a pull_request and it also just contains the branch name.
+env:
+  BRANCH_NAME: ${{ github.head_ref || github.ref_name }}
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "Running tests"
+  tests:
+    name: Running tests
+    runs-on: ubuntu-22.04
+    steps:
+      # Set PHP version to 8.0
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: 8.0
+          coverage: none # disable xdebug, pcov
+          tools: cs2pr
+
+      # We want to reuse our project setup to run different jobs at once.
+      - uses: actions/cache@v3
+        id: opensocial-project-build
+        with:
+          path: ./*
+          key: ${{ github.sha }}
+
+      # Checkout Drupal Social.
+      - uses: actions/checkout@v3
+        with:
+          repository: 'goalgorilla/drupal_social'
+          ref: '5.3.0'
+
+      # Install the version of Open Social under test.
+      - name: Composer require current branch
+        run: composer require goalgorilla/open_social:dev-$BRANCH_NAME
+
+      # Install the docker.
+      - name: Install Docker
+        run: sh scripts/social/ci/install-docker.sh
+
+      - name: Run Docker containers
+        run: docker-compose -f docker-compose-ci.yml up -d
```

Comment:
Can we somehow utilise these things in a cache of sorts? 
It seems these are being re-run every single time, although line 36 shows you're thinking about cache but we never load this or anything. 🤔

Reply:
@ronaldtebrake Cache addition needs to be reverted, they are not adding value in the current setup.

I do have mentioned certain improvements in https://github.com/goalgorilla/open_social/pull/2791 but they have a few challenges as of now. I don't want to block this change. We can do a follow-up later. Please let me know if you think differently.